docker/Makefile

```diff
@@ -38,8 +38,9 @@ all:
 	@echo "    roscube_terminal"
 	@echo "    jetson"
 	@echo "    jetson_terminal"
-	@echo "    unreal_sim"
 	@echo "    zed_camera_driver"
+	@echo "    unreal_sim"
+	@echo "    sim_dev"
```

Comment:
what do we need the sim_dev container for? wouldnt we just use a conda environment like all our other dev stuff?

Reply:
using a conda env doesnt pick up the topics that the unreal sim container outputs

Reply:
... y tho

Reply:
it _should_ be fine